config/core/deployments/activator.yaml

```diff
@@ -44,11 +44,11 @@ spec:
         # https://github.com/knative/serving/issues/1625#issuecomment-511930023
         resources:
```

Comment:
If I run the instructions here: https://github.com/knative/serving/blob/master/DEVELOPMENT.md#deploy-knative-serving won't I get both the Deployment version and DaemonSet version?

Shouldn't you only run with one type?

On a similar note, how will this manifest in the v0.13.0 release instructions?

Reply:
It should not change anything there.
But we need to document probably that we can turn this off.

Reply:
Deployment wise nothing should change. Just execute the standard collective YAML.
But we need to document that you can pick or the other or both.